src/matcher/plugin.py

```diff
@@ -336,4 +383,16 @@ def pytest_addoption(parser) -> None:
       , default=False
       )
 
+
+@pytest.hookimpl(trylast=True)
+def pytest_configure(config: pytest.Config):
+    """Register additional configuration."""
+    return_codes = os.getenv('PYTEST_MATCHER_RETURN_CODES', '').lower() in ('yes', 'true', '1')
+
+    if config.getoption('--pm-reveal-unused-files'):
```

Comment:
No need to check the environment unless CLI option has given
```suggestion
    if config.getoption('--pm-reveal-unused-files'):
        return_codes = os.getenv('PYTEST_MATCHER_RETURN_CODES', '').lower() in ('yes', 'true', '1')
```